weasyprint/css/validation.py

```diff
@@ -1267,6 +1269,23 @@ def bookmark_level(token):
         return 'none'
 
 
+@validator(prefixed=True)  # CSS3 GCPM, experimental
+def string_set(tokens):
+    """Validation for ``string-set``."""
+    if len(tokens) != 2:
+        raise InvalidValues("Only one value is supported. Received %s" %len(tokens)-1)
+    function = parse_function(tokens[1])
+    if function is None:
+        keyword = 'text'
+    else:
+        name, args = function
+        keyword = args[0].value
+    var_name = get_keyword(tokens[0])
+    if tokens[0].type == "IDENT" and keyword in ('text', 'before', 'after'):
+        return ('keyword', keyword, 'name', var_name)
```

Comment:
Do the `'keyword'` and `'name'` strings add information here, or could the return value be just `(keyword, name)`? (Or `(name, keyword)`, to match the order of the syntax?)